claim/schema.py

```diff
@@ -201,8 +203,9 @@ def resolve_claim_admins(
             hf_filters += [Q(location__uuid=district_uuid)]
         elif region_uuid is not None:
             hf_filters += [Q(location__parent__uuid=region_uuid)]
-        if settings.ROW_SECURITY:
-            hf_filters += [LocationManager().build_user_location_filter_query( info.context.user._u, prefix='location') ]
+        #if settings.ROW_SECURITY: already in get_queryset
+        #    hf_filters += [LocationManager().build_user_location_filter_query( info.context.user._u, prefix='location') ]
+
```

Comment:
Why this check was disabled?

Reply:
I will have to double check for this one but down the gql call get_queryset is called, it has already the filter